### PR TITLE
Fix spectrum linedraw, audio sensitivity, tune Stars effect, fix warnings

### PR DESCRIPTION
**Pull request by davepl**
Fix spectrum linedraw, audio sensitivity, tune Stars effect, fix warnings

* [X] I read the contribution guidelines in [CONTRIBUTING.md](http://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/CONTRIBUTING.md).
* [X] I understand the BlinkenPerBit metric, and maximized it in this PR.
* [X] I selected `main` as the target branch.
* [X] All code herein is subjected to the license terms in [COPYING.txt](http://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/COPYING.txt).

### CODE REVIEW

File: include/effects/matrix/spectrumeffects.h [hunk 1]
```diff
@@ -292,11 +292,11 @@ class SpectrumAnalyzerEffect : public LEDStripEffect, virtual public VUMeterEffe
                 float agePercent = (float) msPeakAge / (float) MS_PER_SECOND;
                 uint8_t fadeAmount = std::min(255.0f, agePercent * 256);
                 colorHighlight.fadeToBlackBy(fadeAmount);
```
Comment (davepl) [inline include/effects/matrix/spectrumeffects.h:294]: This was always under-drawing by 1, but once I fixed the Bresenham impl, I had to fix this as it should have been in the first place

File: include/globals.h [hunk 2]
```diff
@@ -730,12 +730,12 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
     #define PROJECT_NAME            "Ledstrip"
     #endif
 
-    #define ENABLE_WEBSERVER            1   // Turn on the internal webserver
```
Comment (davepl) [inline include/globals.h:733]: Make more memory available in LEDSTRIP/DEMO builds

File: include/soundanalyzer.h [hunk 3]
```diff
@@ -85,7 +85,7 @@ void IRAM_ATTR AudioSerialTaskEntry(void *);
 // Simple data class that holds the music peaks for up to 32 bands.  When the sound analyzer finishes a pass, its
 // results are simplified down to this small class of band peaks.
 
-#define MIN_VU 128              // Minimum VU value to use for the span when computing VURatio.  Contributes to
```
Comment (davepl) [inline include/soundanalyzer.h:88]: Fine-tuning overall audio sensitivity

File: src/main.cpp [hunk 4]
```diff
@@ -540,7 +540,7 @@ void loop()
             strOutput += str_sprintf("LED FPS: %d ", g_Values.FPS);
 
             #if USE_STRIP
-                strOutput += str_sprintf("LED Bright: %f, LED Watts: %u, ", g_Values.Brite, g_Values.Watts);
+                strOutput += str_sprintf("LED Bright: %3.0lf%%, LED Watts: %d, ", g_Values.Brite, g_Values.Watts );
```
Comment (rbergen) [inline src/main.cpp:543]: I think the `%d` for `g_Values.Watts` should be a `%u`, because `g_Values.Watts` is a `uint32_t`.

**Review by rbergen — APPROVED** [review]
One small comment on one format specifier, but otherwise LGTM!

**Comment by davepl** [comment]
Fixed locally

File: include/globals.h (hunk 2 above)
Comment (timothom) [reply include/globals.h:733]: So the webserver is no longer  able to be used on LEDSTRIP builds  on a standard  esp32 with 4M memory?

I've been trying to get mine to work and it won't launch.

**Pull request merged**